communication/nats/dialog/dialog_establisher.go

```diff
@@ -67,31 +66,32 @@ func (establisher *dialogEstablisher) EstablishDialog(
 	peerCodec := establisher.newCodecForPeer(peerID)
 
 	peerSender := establisher.newSenderToPeer(peerAddress, peerCodec)
-	err = establisher.negotiateDialog(peerSender)
+	topic, err := establisher.negotiateDialog(peerSender)
```

Comment:
maybe negiotiatTopic? It strange to see dialog negotiation before new dialog creation (the line below)

Reply:
Right, renamed it.